src/main/resources/vertx-groovy/template/groovy.templ

```diff
@@ -176,13 +176,13 @@ this.delegate
 		@else{eventKind == CLASS_JSON_OBJECT}
```

Comment:
This is where the substantive changes are.  There didn't seem to be tests for the generation so I'm not sure how we are supposed to confirm the new generation (this may be what @vietj was referring to).

I will be creating a PR in mongo-client which relies upon these changes (and other groovy generations will be impacted as well, though I'm not sure what the process is for getting those updated).